src/lwc/signals/core.ts

```diff
@@ -463,4 +534,8 @@ function $resource<ReturnType, Params>(
   };
 }
 
-export { $signal, $effect, $computed, $resource };
+function isASignal(anything: unknown): anything is Signal<unknown> {
```

Comment:
![Drop the A. Just isSignal. It's Cleaner](https://github.com/user-attachments/assets/3d174ad0-1e1f-49cf-b58b-364da5af39d7)

Reply:
haha, fair enough